src/reducers/versions.tsx

```diff
@@ -1333,6 +1361,27 @@ export const createReducer = ({
           },
         };
       }
+      case getType(actions.loadVersionFileFromVersion): {
+        const { path, version } = action.payload;
+
+        return {
+          ...state,
+          versionFiles: {
+            ...state.versionFiles,
+            [version.id]: {
```

Comment:
I'm reading this (perhaps through my python sunglasses) as wrapping `version.id` in a list and making the list an object key/property ... which seems a strange thing to do.

Reply:
It makes sense that your python sunglasses would show you that. If you switch those for these JS sunglasses 🕶️ , then you'll see that what's being said here is that the name of the property in question is the value of `version.id`, so if `version.id == 1` then the property would be called `1`.

Reply:
interesting.  I guess the obvious ` version.id: {` wouldn't work because Javascript's syntax is so overloaded.

Reply:
Yeah, I'm pretty sure that's not legal.